invenio_oauthclient/contrib/keycloak/helpers.py

```diff
@@ -53,7 +53,45 @@ def get_public_key(remote, realm_url):
     return certs_resp["public_key"]
 
 
-def get_user_info(remote, resp_token, fallback_to_endpoint=True, options=dict()):
+def _get_user_info_from_token(remote, token, config_prefix):
+    """Get the user information from the JWT token."""
+    # try to parse the "id_token" part of Keycloak's (JWT) response
+    realm_url = current_app.config[f"{config_prefix}_REALM_URL"]
+    pubkey = _format_public_key(get_public_key(remote, realm_url))
+    alg = jwt.get_unverified_header(token)["alg"]
+
+    if not isinstance(options, dict):
```

Comment:
won't this crash because `options` isn't defined here yet?
just setting `options = {}` should be fine